openvdb/openvdb/version.h.in

```diff
@@ -6,83 +6,85 @@
 /// where N is the ABI version number.
-/// The ABI version must be set consistently when building code that depends on OpenVDB.
 ///
-/// The ABI version number defaults to the library major version number,
-/// which gets incremented whenever changes are made to the ABI of the
-/// Grid class or related classes (Tree, Transform, Metadata, etc.).
-/// Setting the ABI version number to an earlier library version number
-/// disables grid ABI changes made since that library version.
-/// The OpenVDB 1.x ABI is no longer supported, and support for other old ABIs
-/// might also eventually be dropped.
+/// The ABI version must be set consistently when building code that depends on
+/// OpenVDB.
+///
+/// The ABI version number defaults to the library major version number, which
+/// gets incremented whenever changes are made to the ABI of the Grid class or
+/// related classes (Tree, Transform, Metadata, etc.). Setting the ABI version
+/// number to an earlier library version number disables grid ABI changes made
+/// since that library version.
 ///
 /// The library minor version number gets incremented whenever a change is made
 /// to any aspect of the public API (not just the grid API) that necessitates
 /// changes to client code.  Changes to APIs in private or internal namespaces
-/// do not trigger a minor version number increment; such APIs should not be used
-/// in client code.
+/// do not trigger a minor version number increment; such APIs should not be
+/// used in client code.
 ///
-/// A patch version number increment indicates a change&mdash;usually a new feature
-/// or a bug fix&mdash;that does not necessitate changes to client code but rather
-/// only recompilation of that code (because the library namespace incorporates
-/// the version number).
+/// A patch version number increment indicates a change&mdash;usually a new
+/// feature or a bug fix&mdash;that does not necessitate changes to client code
+/// but rather only recompilation of that code (because the library namespace
+/// incorporates the version number).
 ///
-/// The file format version number gets incremented when it becomes possible
-/// to write files that cannot safely be read with older versions of the library.
-/// Not all files written in a newer format are incompatible with older libraries, however.
-/// And in general, files containing grids of unknown type can be read safely,
-/// although the unknown grids will not be accessible.
+/// The file format version number gets incremented when it becomes possible to
+/// write files that cannot safely be read with older versions of the library.
+/// Not all files written in a newer format are incompatible with older
+/// libraries, however. And in general, files containing grids of unknown type
+/// can be read safely, although the unknown grids will not be accessible.
 
 #ifndef OPENVDB_VERSION_HAS_BEEN_INCLUDED
 #define OPENVDB_VERSION_HAS_BEEN_INCLUDED
 
-#include "Platform.h"
+#include "openvdb/Platform.h"
```

Comment:
Because this file is auto-generated into the `build`, "Platform.h" doesn't exist there.